src/sentry/lang/java/utils.py

```diff
@@ -32,12 +34,14 @@ def get_proguard_images(event: Event):
 
 
 def get_proguard_mapper(uuid: str, project: Project):
-    with sentry_sdk.start_span(op="proguard.get_proguard_mapper"):
+    with sentry_sdk.start_span(op="proguard.get_proguard_mapper") as span:
         dif_paths = ProjectDebugFile.difcache.fetch_difs(project, [uuid], features=["mapping"])
         debug_file_path = dif_paths.get(uuid)
         if debug_file_path is None:
             return
 
+        proguard_file_size_in_mb = os.path.getsize(debug_file_path) / (1024 * 1024.0)
```

Comment:
Should we wrap this in a try catch just in case? I think `os.path.getsize` might throw an error if it cant reach the file for whatever reaason

Reply:
oh good point, I'll do that. Thanks for the suggestion